examples/server/main.go

```diff
@@ -286,3 +287,29 @@ func DocumentsHandler(w http.ResponseWriter, r *http.Request) {
 		Documents: documents,
 	})
 }
+
+// ShiftsHandler is the handler used for get all the shifts
+// and print them on a list template
+func ShiftsHandler(w http.ResponseWriter, r *http.Request) {
+	cl, err := factorial.New(
+		factorial.WithOAuth2Client(provider.Client(token)),
+	)
+	if err != nil {
+		log.Panic(err)
+	}
+
+	shifts, err := cl.ListShifts(nil)
+	if err != nil {
+		log.Panicln("Error while getting shifts", err)
+	}
+	log.Println("[DEBUG] shifts", len(shifts))
```

Comment:
Maybe we can remove this log debug after be used haha

Reply:
Yep, I think we can clean-up a bit the server once we finish all endpoints, for now let it there bc it helps to debug the response just in case the template is not working untill we know its working